src/site/content/en/blog/granular-chunking-nextjs/index.md

```diff
@@ -179,8 +179,7 @@ getDependencies (route) {
 ```
 
 <figure class="w-figure">
-  <img class="w-screenshot-filled" src="outputted-chunks.png" 
-       alt="Output of multiple shared chunks in a Next.js application.">
+  <img class="w-screenshot-filled" src="outputted-chunks.png" alt="Output of multiple shared chunks in a Next.js application.">
```

Comment:
Missed one? Also looks like it should be using `w-screenshot w-screenshot--filled`